internal/autocert/manager.go

```diff
@@ -166,7 +166,7 @@ func (mgr *Manager) renewConfigCerts(ctx context.Context) error {
 	for _, domain := range sourceHostnames(cfg) {
 		cert, err := cm.CacheManagedCertificate(domain)
 		if err != nil {
-			log.Error(ctx).Err(err).Str("domain", domain).Msg("get cert")
+			// this happens for unmanaged certificates
```

Comment:
Should we handle the error if `cert` is nil just to be safe? Or is that impossible?

https://github.com/caddyserver/certmagic/blob/b74a9ce8088fde76facb32d545d2494f52b227fe/certificates.go#L110

Reply:
If the library follows Go conventions `cert` should never be nil. But I don't know if it does or not, so I guess I can add code to handle this.

Reply:
Nevermind. `cert` is not a pointer type.